pkg/store/event/event.go

```diff
@@ -44,15 +44,135 @@ type eventStore struct {
 	db *dbstore.DB
 }
 
-// TODO: implement core.EventStore interface
-func (e *eventStore) List(context.Context) ([]*core.Event, error) { return nil, nil }
-func (e *eventStore) ListByExperiment(context.Context, string, string) ([]*core.Event, error) {
-	return nil, nil
+// FindPodRecordsByEventID return the list of PodRecords according to the eventID
+func (e *eventStore) FindPodRecordsByEventID(_ context.Context, id uint) ([]*core.PodRecord, error) {
```

Comment:
If this function won't be used by the other pkg, we can change it to `findPodRecordsByEventID`